doc/samples.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to Azure DevOps extension samples
+
+1. [Getting Started]()
+2. [Configuring Git Aliases]()
+3. [Using Azure DevOps behind a proxy]()
+4. [Querying and filtering output]()
+
+## Azure Pipelines
+1. [View and trigger a build]()
```

Comment:
These are WIPs?

Reply:
I was thinking this as the structure for tackling sample documentation.